pubsub/pubsub.go

```diff
@@ -0,0 +1,260 @@
+// Copyright 2018 The Go Cloud Authors
```

Comment:
This file doesn't look like it belongs in this PR.

Reply:
Done.